code/modules/urist/modules/newtrading/trade/trade_items/stock_parts.dm

```diff
@@ -0,0 +1,119 @@
+/datum/trade_item/stock_parts
+	quantity = 8
+	trader_weight = 1
+	value = 160
```

Comment:
i've mentioned before, but i don't like just assigning the same value to all basic stock parts. these prices will also affect the prices for trading being done through cargo as well, so they need a little more consideration than a flat number. I'd lean towards letting get_worth handle it and leaving the value null, but i'm not sure what get_worth returns for these objects though.

Reply:
what would that look like? just: get_worth = what ever?